crates/check/src/lib.rs

```diff
@@ -13,14 +13,14 @@
 //!
 //! ## Solution Validation
 //!
-//! - [`solution::check`] validates an unsigned solution.
-//! - [`solution::check_data`] validates a solution's data slice.
-//! - [`solution::check_state_mutations`] validates a solution's state mutation slice.
+//! - [`solution::check_set`] validates a solution set.
```

Comment:
Is it relevant whether the solution set is signed or not here?

Reply:
Not any longer, we used to have signed solutions long ago but there's no longer any signing of solutions or contracts in-protocol